components/support/nimbus-fml/src/parser.rs

```diff
@@ -426,6 +429,7 @@ impl<'object> DefaultsMerger<'object> {
     }
 
     fn collect_object_defaults(&self, nm: &str) -> Result<serde_json::Value> {
+        #[allow(deprecated)]
```

Comment:
I think just go ahead and remove this deprecated code and property. We can always add it back in later.